Godep.json contains two etcd versions

Current master a86ef13e1e70f9b74ae2b41e392b4f4a61db8e46 has two versions of etcd:
```
$ jq -r '.Deps[] | select(.ImportPath | startswith("github.com/coreos/etcd")) | .Rev' Godeps/Godeps.json | sort -u
83347907774bf36cbb261c594a32fd7b0f5dd9f6
8ba2897a21e4fc51b298ca553d251318425f93ae
```

It breaks godep restore:
```
$ ./hack/godep-restore.sh 
HEAD is now at e36e043... Fix doc version
fatal: remote github.com/openshift-remote already exists.
fatal: remote github.com/openshift-remote already exists.
fatal: remote github.com/openshift-remote already exists.
fatal: remote github.com/openshift-remote already exists.
fatal: remote github.com/openshift-remote already exists.
fatal: remote github.com/openshift-remote already exists.
fatal: remote github.com/openshift-remote already exists.
fatal: remote github.com/openshift-remote already exists.
fatal: remote github.com/openshift-remote already exists.
HEAD is now at 07b16b6... Merge pull request #121 from KevinMGranger/fix/url_prefix_slash
HEAD is now at bd3c8e8... Add extra methods to control the number of times a call is expected.
HEAD is now at eb47ba8... metadata: use ctxhttp to support cancelation in Go 1.4
Starting to download all godeps. This takes a while
godep: error restoring dep (github.com/coreos/etcd/integration): Wanted to restore rev 83347907774bf36cbb261c594a32fd7b0f5dd9f6, already restored rev 8ba2897a21e4fc51b298ca553d251318425f93ae for another package in the repo
godep: error restoring dep (github.com/coreos/etcd/proxy/grpcproxy): Wanted to restore rev 83347907774bf36cbb261c594a32fd7b0f5dd9f6, already restored rev 8ba2897a21e4fc51b298ca553d251318425f93ae for another package in the repo
godep: error restoring dep (github.com/coreos/etcd/proxy/grpcproxy/cache): Wanted to restore rev 83347907774bf36cbb261c594a32fd7b0f5dd9f6, already restored rev 8ba2897a21e4fc51b298ca553d251318425f93ae for another package in the repo
godep: Error restoring some deps. Aborting check.
[ERROR] PID 14092: hack/godep-restore.sh:77: `"${GODEP}" restore` exited with status 1.
[INFO] 		Stack Trace: 
[INFO] 		  1: hack/godep-restore.sh:77: `"${GODEP}" restore`
[INFO]   Exiting with code 1.
```

--- a/Godeps/Godeps.json
+++ b/Godeps/Godeps.json
@@ -563,8 +563,8 @@
 		},
 		{
 			"ImportPath": "github.com/coreos/etcd/integration",
-			"Comment": "v3.1.0-rc.0",
-			"Rev": "83347907774bf36cbb261c594a32fd7b0f5dd9f6"
+			"Comment": "v3.1.0",
+			"Rev": "8ba2897a21e4fc51b298ca553d251318425f93ae"
 		},
 		{
 			"ImportPath": "github.com/coreos/etcd/lease",
@@ -698,13 +698,13 @@
 		},
 		{
 			"ImportPath": "github.com/coreos/etcd/proxy/grpcproxy",
-			"Comment": "v3.1.0-rc.0",
-			"Rev": "83347907774bf36cbb261c594a32fd7b0f5dd9f6"
+			"Comment": "v3.1.0",
+			"Rev": "8ba2897a21e4fc51b298ca553d251318425f93ae"
 		},
 		{
 			"ImportPath": "github.com/coreos/etcd/proxy/grpcproxy/cache",
-			"Comment": "v3.1.0-rc.0",
-			"Rev": "83347907774bf36cbb261c594a32fd7b0f5dd9f6"
+			"Comment": "v3.1.0",
+			"Rev": "8ba2897a21e4fc51b298ca553d251318425f93ae"
 		},
 		{
 			"ImportPath": "github.com/coreos/etcd/raft",

